minet/twitter/api_scraper.py

```diff
@@ -258,7 +262,19 @@ def acquire_guest_token(self):
         guest_token = extract_guest_token(response.data)
```

Comment:
Does this mean the "traditional" method still works sometimes? Should we switch completely to the new method to avoid a potential useless call?

Reply:
I don't know for sure. In my experience/tests this method always fails but some user-agent or else conditions could make it work maybe? Snscrape kept it but I have to say I didn't try to make it work.